index.bs

```diff
@@ -789,35 +792,64 @@ When the {{getSettings()}} method is invoked on a video stream track, the user a
 
     <li><dfn>Focus distance</dfn> is a numeric camera setting that controls the focus distance of the lens.  The setting usually represents distance in meters to the optimal focus distance.</li>
 
-    <li><dfn>Pan</dfn> is a numeric camera setting that controls the pan of the camera. The setting represents pan in arc seconds, which are 1/3600th of a degree. Values are in the range from –180\*3600 arc seconds to +180\*3600 arc seconds. Positive values pan the camera clockwise as viewed from above, and negative values pan the camera counter clockwise as viewed from above.
+    <li>
+      <dfn>Pan</dfn> is a numeric camera setting that controls the pan of the camera. The setting represents pan in arc seconds, which are 1/3600th of a degree. Values are in the range from -180\*3600 arc seconds to +180\*3600 arc seconds. Positive values pan the camera clockwise as viewed from above, and negative values pan the camera counter clockwise as viewed from above.
 
-    Any algorithm which uses a {{MediaTrackConstraintSet}} object and its {{MediaTrackConstraintSet/pan}} dictionary member which exists after a possible normalization MUST <a>request permission to use</a> (as defined in [[!permissions]]) a PermissionDescriptor with its name member set to {{PermissionName/camera}} and its {{CameraDevicePermissionDescriptor/panTiltZoom}} member set to true, and, optionally, consider its {{DevicePermissionDescriptor/deviceId}} member set to any appropriate device's deviceId.
+      <dfn>Pan capability</dfn> is a boolean camera property representing camera's ability to pan which results in a proper non-degenerate setting range.
+      That capability or whether it is exposed cannot be changed with {{applyConstraints()}}.
 
-    If the {{visibilityState}} of the <a>top-level browsing context</a> value is "hidden", the {{applyConstraints()}} algorithm MUST throw a {{SecurityError}} if {{MediaTrackConstraintSet/pan}} dictionary member exists after a possible normalization.
+      If the {{getUserMedia()}} algorithm uses a {{MediaTrackConstraintSet}} object and its {{MediaTrackConstraintSet/pan}} dictionary member which exists after a possible normalization:
```

Comment:
"dictionary member exists after normalization"?

The word "which" seems unnecessary and I don't understand the significance of the word "possible".

Reply:
So there are actually three conditions:

1. the `getUserMedia()` algorithm uses a `MediaTrackConstraintSet` object
2. the `getUserMedia()` algorithm uses the `pan` dictionary member of the `MediaTrackConstraintSet` object
3. the dictionary member exists after normalization

Dropping the word "which" would drop the second condition. I spelled out all three condition, instead. I hope it is more clear now.

I previously used the word "possible" because normalization is done only for boolean values but not for non-existing values, for double values nor for ConstrainDoubleRange values. It is not significant so I dropped it.